statsd/end_to_end_udp_test.go

```diff
@@ -9,6 +9,12 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func dummyContainerID() string { return "" }
+
+func init() {
+	containerID = dummyContainerID
+}
+
```

Comment:
This will impact every single test of the package, which means that by default we won't test the real code.